### ISSUE TITLE
Using newer versions of Electron(9.0.0 and up)

### ISSUE THREAD

**Issue by KorbsStudio** [simply-coded/electron-navigation#34]
When using later versions of Electron, nothing shows on the screen.

**Comment by CleverCarpet** [comment]
Same issue, tried using older version too

**Comment by traderjosh** [comment]
Yeah, it worked with a very old version, nothing showing on the screen. Any suggestions on how to debug?

**Comment by traderjosh** [comment]
I could only get it to work on electron version 4. Haven't tested it on versions above, but people say it doesn't work on versions above 9. Would like to get it to work with higher versions so I can keep the electron version up to date.
